CMakeLists.txt

```diff
@@ -35,8 +35,8 @@ include(CheckCXXCompilerFlag)
 include(cmake/git_version.cmake)
 set_git_version(CMAKE_PROJECT_VERSION)
 
-# Set default standard to C++20
-set(CMAKE_CXX_STANDARD_MIN 20)
+# Set default standard to C++17
+set(CMAKE_CXX_STANDARD_MIN 17)
```

Comment:
I'm wondering if we should keep the minimum at C++20. Already before the Acts and ROOT upgrades to C++20, we were using C++20 features in EICrecon.

Reply:
Removed this change and keeping at C++20.